### DISCUSSION TITLE
Issue upgrading to v2 from v1, having error 'Cannot read property 'nbind' of undefined'

### DISCUSSION THREAD

**Discussion by dev-scb** [diegomura/react-pdf#1342] [Q&A]
Hi,

I am getting console errors just by trying to upgrade my library from v1.6.12 to v2.0.13 
It works well if I do not upgrade and stay on v1.6.12

in my package.json:
{
 "@react-pdf/renderer": "^2.0.13",
  "@react-pdf/styled-components": "^1.4.0",
...
"react": "17.0.1",
      "react-dom": "17.0.1",
"webpack": "^4.43.0",
    "webpack-cli": "^3.3.11",
    "webpack-dev-server": "^3.10.3",
    "webpack-merge": "^4.2.2"
}

Windows 10, Chrome browser, React 17, Webpack 4

Screenshot:
![image](https://user-images.githubusercontent.com/85145010/120259844-43ff0900-c2c7-11eb-9d58-42f89762ac73.png)

Please help?

**Comment by diegomura** [comment]
`@react-pdf/styled-components` is deprecated. I don't think that's the cause of this though but I recommend just using the `Stylesheet` API. I can't say why this is happening based on that data. Is there a way I can replicate it?

**Reply by dev-scb** [reply]
My current repo is large, so I'll have to first find a way to recreate a demo and extract relevant configs which can reproduce that error.
Was hoping it was a known issue but seems like it maybe due to some obscure config setting, i'll try to dig deeper at my end here first.

Thanks for the reply, diegomura.
